testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js

```diff
@@ -754,6 +754,31 @@ QUnit.module("options changing", moduleConfig, () => {
         }).trigger(dxPointerDown);
     });
 
+    QUnit.test("tap on clear button should not raise onValueChange event (T812448)", (assert) => {
+        let callCount = 0;
```

Comment:
`const valueChangeStub = sinon.stub`
<...>
`onValueChanged: valueChangeStub`
<...>
You can check `valueChangeStub.callCount`(number) or `valueChangeStub.calledOnce`(bool)

Reply:
Thanks, value change counter is replaced by stub now.